rest_api.py

```diff
@@ -101,10 +103,22 @@ def get(self):
     return json.dumps(response)
 
 
+class AirflowDB(Resource):
+
+  def get(self):
+    parser = reqparse.RequestParser()
+    parser.add_argument('cm')
+    args = parser.parse_args()
+    data = query_airflow(str(args['cm']))
```

Comment:
how is the query constructed?

Reply:
What exactly do you mean?

Reply:
Who construct the SQL queries? airflow adapter?
And how does it derive filtering, grouping, and ordering conditions?

Reply:
Currently an airflow adapter has not been implemented, though eventually the airflow adapter will be responsible for constructing the airflow queries. The /airflowdb request handler which is currently implemented is there more for manual SQL queries rather than use by the server itself. (I implemented this initially because it can be very difficult to test whether things are working as expected in App Engine by itself, and this was a simple addition for me)

In the future I do not plan on using /airflowdb to handle requests from the frontend at all. Instead I plan to have /releases, /release, and /tasks all call the airflow adapter individually, and will pass in the requisite parameters which will allow the adapter to construct the SQL query. These HTTP methods will then return the requested data which fits the parameters